knowledge-base/how-to-split-a-nested-tabel-on-two-pages.md

```diff
@@ -0,0 +1,42 @@
+---
+title: How to split a nested table on two pages
+description: A common requirement when using nested tables in the report is to force them to break between pages. 
```

Comment:
Avoid fluff in the metatags. For example "A common requirement when using nested tables ...." should become something like: "How to force nested table to break between pages."